### ISSUE TITLE
[BUG]: FAILED TO FETCH

### ISSUE THREAD

**Issue by thistleknot** [Mintplex-Labs/anything-llm#2293]
### How are you running AnythingLLM?

Docker (local)

### What happened?

An error occurred while streaming response. network error
embedder
![image](https://github.com/user-attachments/assets/00cb1366-db03-4665-9904-75cceb7586ca)

my server produced embeddings (and the same embedded that embedded the documents)
![image](https://github.com/user-attachments/assets/17202994-c3a5-45cb-8350-cc92651c10af)

### Are there known steps to reproduce?

windows 10
setup with text-generation-webui openai proxy

same error happens when I use anythingllm's embedder as well (starting a new workspace).

**Comment by thistleknot** [comment]
tried setting the model to openai with appropriate key.

Issue appears to be with embeddings?
![image](https://github.com/user-attachments/assets/cbad0e88-c4ab-418d-9a59-6931bb3357e5)

![image](https://github.com/user-attachments/assets/379b9708-8770-46df-adf6-72c43a40c2e5)

**Comment by thistleknot** [comment]
Reinstalling as all users

**Comment by thistleknot** [comment]
![image](https://github.com/user-attachments/assets/3638ba41-d503-4776-98b8-bb39eae2d0c2)

**Comment by timothycarambat** [comment]
Issues for the uncatchable generic fetch failed:
- You do not have Visual C++ dist installed (windows only & default vectorDB as selection): https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#latest-microsoft-visual-c-redistributable-version

- You cannot download from hugginface.co/aws s3 for the embedding model. You show using OpenAI embedder, but the logs show native embedder is used so this issue/restriction could apply.
https://github.com/Mintplex-Labs/anything-llm/blob/0deb0a01c68d97d4da4885c0239dfbd713e33bf4/server/utils/EmbeddingEngines/native/index.js#L11

- (Rare case & using default vector db only): Your CPU does not support AVX2 instruction set.

**Comment by thistleknot** [comment]
My cpu doesn't support avx2

**Comment by thistleknot** [comment]
Would also be nice to know what exact version of vc++ I'd required if
necessary

**Comment by thistleknot** [comment]
Can you explain why I would need to know why I can't download from a particular aws point if I don't set that setting explicitly?

**Comment by timothycarambat** [comment]
> Would also be nice to know what exact version of vc++ I'd required if

Just the latest, but pretty sure its 14 IIRC.

> Can you explain why I would need to know why I can't download from a particular aws point if I don't set that setting explicitly?

If the HuggingFace download endpoint is blocked because of firewall, or more commonly - your geography or VPN, we fallback to a duplicate of the files on an AWS bucket we host as a convenience for those who are blocked by HF for whatever reason.

**Comment by thistleknot** [comment]
I'm not blocked by hf. Wasn't sure if I needed to specify an hf token.

Thanks for clarifying. Unsure what there is I can do for windows no avx2
but I'm okay. I suppose wsl if i really cared as I said I got it working on
ol8 on same network
